crates/matrix-sdk-sqlite/migrations/state_store/007_a_send_queue_wedge_reason.sql

```diff
@@ -0,0 +1,4 @@
+-- New send queue events, now persists the type of error causing it to be wedged
+ALTER TABLE "send_queue_events"
+    -- Used as a value, thus encrypted/decrypted
+    ADD COLUMN "wedge_reason" BLOB;
```

Comment:
I don't know how this works in Rust, would this still work if `QueueWedgeError` adds/removes some case?

Reply:
Yeah, the value is serialized (I suspect to JSON bytes) before being put into the table, and it has to be a blob because of the encryption case.

Reply:
Is there no need to annotate this to mark it nullable? and that the default value is null when not set? Maybe it's the default with sqlite, and I don't know, but might be worth indicating in the comment then?

Reply:
Added some comments. And yes the default in NULLABLE and NULL.